heron/spi/src/java/BUILD

```diff
@@ -59,6 +59,7 @@ utils_deps_files = \
 scheduler_deps_files = \
     heron_java_proto_files() + [
         ":common-spi-java",
+        ":packing-spi-java",
```

Comment:
The rule `scheduler-spi-java` depends on the PackingPlan.java class and the scheduler_deps_files is used to indicate the dependency for this rule.